SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -39,6 +39,17 @@ DEFINE_LOG_CATEGORY(LogSpatialOSNetDriver);
 DECLARE_CYCLE_STAT(TEXT("ServerReplicateActors"), STAT_SpatialServerReplicateActors, STATGROUP_SpatialNet);
 DEFINE_STAT(STAT_SpatialConsiderList);
 
+USpatialNetDriver::USpatialNetDriver(const FObjectInitializer& ObjectInitializer)
+	: Super(ObjectInitializer)
+	, bAuthoritativeDestruction(false)
```

Comment:
Possible I've misunderstood, but this is always set to true in the init function. Is that function always called?

Reply:
It is always called, I just added a initializer list to default to sane values. I'll switch to true.